fala/templates/adviser/results.html

```diff
@@ -0,0 +1,135 @@
+          Showing {{ page_count }} results around
+          {% pluralize %}
+          {{ result_count }} results in total. Showing {{ page_count }} results in order of closeness to
+        {% endtrans %}
+        <strong class="notranslate" translate="no">{{ data.origin.postcode }}</strong>
+        {%- if form.name.value() %}
+          {{ _('matching') }} <strong>{{ form.name.value() }}</strong>.
+        {%- else -%}
+          .
+        {% endif %}
+      {% else %}
+        {% trans result_count=data.count, page_count=data.results|length, org_name=form.name.value() %}
+          Showing {{ page_count }} result for <strong>{{ org_name }}</strong>.
+          {% pluralize %}
+          {{ result_count }} results in total. Showing {{ page_count }} results for <strong>{{ org_name }}</strong>.
+        {% endtrans %}
+      {% endif %}
+      <hr/>
+      <div class="org-list">
+        {% for item in data.results %}
+          <div class="org-list-item results-list-item" data-lat="{{ item.location.point.coordinates.1 }}" data-lon="{{ item.location.point.coordinates.0 }}" data-id="{{ loop.index }}">
+            <div class="org-details">
+              {% if item.distance %}
+                <p class="miles-away govuk-body">
+                  <span class="govuk-visually-hidden">{{ _('Distance') }}</span>
+                  {% trans miles=item.distance|round(2) %}{{ miles }} miles away{% endtrans %}
+                </p>
+              {% endif %}
+              <span class="fn org govuk-body-l govuk-!-font-weight-bold">{{ item.organisation.name }}</span>
+              <p class="govuk-body">
+                <span>{{ _('Telephone') }}:</span>
+                <span class="tel">{{ item.telephone }}</span>
+              </p>
+              <p class="govuk-body govuk-!-margin-top-0">
+                <span>{{ _('Address') }}:</span>
+                <span class="govuk-visually-hidden">{{ _('Address') }}:</span>
+                <span class="adr">
+                        <span class="street-address notranslate" translate="no">{{ item.location.address }}</span>
+                        <span class="city">{{ item.location.city }}</span>
+                        <span class="postal-code notranslate" translate="no">{{ item.location.postcode }}</span>
+                      </span>
+              </p>
+              {% if item.organisation.website %}
+                <p class="govuk-body">
+                  <span>{{ _('Website') }}:</span>
+                  <a class="govuk-link" href="{{ item.organisation.website|human_to_url }}" class="url" target="_blank" rel="noopener">
+                    {{ item.organisation.website|url_to_human + " (Opens in new tab)" }}
+                  </a>
+                </p>
+              {% endif %}
+              {% if item.categories|length %}
+                <span class="govuk-heading-s govuk-!-margin-bottom-2">{{ _('Categories of law') }}</span>
```

Comment:
I really sorry, for `Categories of law` header, I told you to make this a span HTML. But actually it can be a H3